recipes/h3ronpy/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.22.0" %}
+
+package:
+  name: h3ronpy
+  version: {{ version }}
+
+source:
+  url: https://github.com/nmandery/h3ronpy/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d31d41e91191ff4652e6d9d61939eebfaec0b0dccd89c71ac52f2ca8d9a1e6c5
+
+build:
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install ./h3ronpy -vv
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - maturin >=1.7.0,<2.0                   # [build_platform != target_platform]
+    - cargo-bundle-licenses
+    - {{ stdlib("c") }}
+    - {{ compiler("c") }}
+    - {{ compiler("rust") }}
+  host:
+    - python
+    - maturin >=1.7.0,<2.0
+    - pip
+  run:
+    - python
+    - arro3-core >=0.4
+    - numpy >=1.24
+  run_constrained:
+    - geopandas >=1
+    - pyarrow >=15
+    - polars >=1
+
+test:
+  imports:
+    - h3ronpy
+    - h3ronpy.pandas
+    - h3ronpy.pandas.raster
+    - h3ronpy.pandas.vector
+    - h3ronpy.polars
+  commands:
+    - pip check
+  requires:
+    - pip
+    - geopandas
+    - pyarrow
+    - polars
```

Comment:
Either make then `run` dependencies or omit the optional modules from the import check
```suggestion
```